docs/source/python/api/tables.rst

```diff
@@ -29,6 +29,7 @@ Factory Functions
    :toctree: ../generated/
 
    chunked_array
+   combine_chunks
```

Comment:
concat_arrays ?

Reply:
Added

Reply:
`combine_chunks` is a method, and not a function, so adding it here won't work (the method will automatically be added to the ChunkedArray docstring page, though)